src/routes.php

```diff
@@ -267,4 +267,17 @@
     )
 ));
 
+// $map_routes = include __DIR__ . '/Routes/map_routes.php';
+// $map_routes->addPrefix('/map');
+// $routes->addCollection($map_routes);
```

Comment:
@davidbeig could these commented lines be removed?

Reply:
We could also move the previously defined routes inside this map_routes.php.